src/modal/components/ItemModal.jsx

```diff
@@ -13,33 +13,53 @@ const Input = styled.input`
   margin-bottom: 32px;
 `;
 
+// const itemAddedModal = styled.div`
+// display: none;
+// border: 1px solid #bcbcbc;
+// background: green;
+// color: red;
+// z-index: 20;
+// `;
+
 const DeleteIcon = styled(BaseDeleteIcon)`
   width: 32px;
   height: 32px;
   color: #f11a1a;
+  cursor: pointer;
 `;
 
+
+
 const ItemModal = ({ item = {}, onClose, onSave, onDelete }) => {
   const { t } = useTranslation();
   const [state, setState] = useState({ type: "unit", ...item });
   const [showDeleteModal, setShowDeleteModal] = useState(false);
 
   const categoriesOptions = t("categories", { returnObjects: true });
 
+  const onSaveItem = () => {
+    if (!state.name || state.amount === undefined){
+      return;
+    }
+    onSave(state)
+  }
+
   return (
     <Modal
       onClose={onClose}
       title={item.id ? t("update_item") : t("add_item")}
       primaryText={t("save")}
       onPrimaryClick={() => {
-        if (!state.name || state.amount === undefined) {
-          return;
-        }
-        onSave(state);
+        onSaveItem();
         onClose();
       }}
       secondaryText={t("cancel")}
       onSecondaryClick={onClose}
+      thirdText={t("add_another_item")}
+      onThirdClick={() => {
+        onSaveItem();
+        setState({});
```

Comment:
The initial state is {type: unit} (line 35). After adding the first item - it should keep the same behaviour (default type -unit)